doc/start/quick-cephfs.rst

```diff
@@ -3,117 +3,115 @@
-
-#. Identify a key for a user within a keyring file. For example:: 
+    ceph osd pool set data size {number-of-osds}
+    ceph osd pool set meta size {number-of-osds}
 
-	cat ceph.client.admin.keyring
+And, in case you have deployed all of the OSDs on the same node, you would need
+to create a new CRUSH rule to replicate data across OSDs and set the rule on the
+CephFS pools, since the default CRUSH rule is to replicate data across
+different nodes::
 
-#. Copy the key of the user who will be using the mounted CephFS file system.
-   It should look something like this:: 
-	
-	[client.admin]
-	   key = AQCj2YpRiAe6CxAA7/ETt7Hcl9IyxyYciVs47w==
+    ceph osd crush rule create-replicated {rule_name} default osd
+    ceph osd pool set data crush_rule (rule_name} data
+    ceph osd pool set data crush_rule {rule_name} meta
 
-#. Open a text editor. 
+Usually, setting ``pg_num`` to 32 gives a perfectly healthy cluster. If you need
+to pick appropriate value for ``pg_num`` based on your needs, refer
+`Placement Group`_ reference. Nautilus release introduced pg_autoscaler plugin
+which can automatically increase/decrease value of ``pg_num``; to use it refer
+the `Placement Group`_ reference.
 
-#. Paste the key into an empty file. It should look something like this::
-
-	AQCj2YpRiAe6CxAA7/ETt7Hcl9IyxyYciVs47w==
-
-#. Save the file with the user ``name`` as an attribute 
-   (e.g., ``admin.secret``).
-
-#. Ensure the file permissions are appropriate for the user, but not
-   visible to other users. 
-
-
-Kernel Driver
-=============
+Mounting the File System
+========================
 
-Mount CephFS as a kernel driver. ::
+Using Kernel Driver
+-------------------
 
-	sudo mkdir /mnt/mycephfs
-	sudo mount -t ceph {ip-address-of-monitor}:6789:/ /mnt/mycephfs
+The command to mount CephFS using kernel driver looks like this::
 
-The Ceph Storage Cluster uses authentication by default. Specify a user ``name``
-and the ``secretfile`` you created  in the `Create a Secret File`_ section. For
-example::
+    sudo mount -t ceph :{path-to-mounted} {mount-point} -o name={user-name}
+    sudo mount -t ceph :/ /mnt/mycephfs -o name=admin   # usable version
 
-	sudo mount -t ceph 192.168.0.1:6789:/ /mnt/mycephfs -o name=admin,secretfile=admin.secret
+``{path-to-be-mounted}`` is the path within CephFS that will be mounted,
+``{mount-point}`` is the point in your file system upon which CephFS will be
+mounted and ``{user-name}`` is the name of CephX user that has the
+authorization to mount CephFS on the machine. Following command is the fuller
+form, however these extra details are automatically figured out by
+the kernel driver::
 
+    sudo mount -t ceph {ip-address-of-MON}:{port-number-of-MON}:{path-to-be-mounted} -o name={user-name},secret={secret-key} {mount-point}
 
-.. note:: Mount the CephFS file system on the admin node,
-   not the server node. See `FAQ`_ for details.
+Using FUSE
+----------
 
+To mount CephFS using file system in User Space (FUSE) run::
```

Comment:
```suggestion
To mount CephFS using File System in User Space (FUSE) run::
```

Reply:
@rishabh-d-dave I notice that we write FIle System instead of FIlesystem everywhere but I'm not so sure we should do this for the expansion of FUSE as it is defined to be Filesystem in Userspace: http://man7.org/linux/man-pages/man4/fuse.4.html

Reply:
Yeah, probably we shouldn't. @ceph/cephfs any opinion?

Reply:
Changed it to "Filesystem in User Space" for now.